nio_cli/commands/blockcheck.py

```diff
@@ -0,0 +1,241 @@
+import json
+import os
+import sys
+import subprocess
+
+from .base import Base
+
+
+class BlockCheck(Base):
+
+    # This should be run from inside the block dir
+
+    def __init__(self, options, *args, **kwargs):
+
+        super().__init__(options, *args, **kwargs)
+        self._block = os.getcwd().split('/')[-1]
+        self.all_contents = os.listdir('.')
+        self.block_files_including_base = [
+            f for f in self.all_contents if '.py' in f
+        ]
+        self.block_files = [
+            f for f in self.all_contents if 'block.py' in f and 'base' not in f
```

Comment:
Not sure that it would be safe to just avoid all `base` files. Is that even a standard that we stick to? Maybe we should.